internal/resources/metal/organization_member/models.go

```diff
@@ -0,0 +1,67 @@
+package organizationmember
+
+import (
+	"context"
+	"path"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+type ResourceModel struct {
+	ID             types.String `tfsdk:"id"`
+	Invitee        types.String `tfsdk:"invitee"`
+	InvitedBy      types.String `tfsdk:"invited_by"`
+	OrganizationID types.String `tfsdk:"organization_id"`
+	ProjectsIDs    types.Set    `tfsdk:"projects_ids"`
+	Nonce          types.String `tfsdk:"nonce"`
+	Message        types.String `tfsdk:"message"`
+	Created        types.String `tfsdk:"created"`
+	Updated        types.String `tfsdk:"updated"`
+	Roles          types.Set    `tfsdk:"roles"`
+	State          types.String `tfsdk:"state"`
+}
+
+// func (m *ResourceModel) parse(ctx context.Context, invitee string, members []packngo.Member, invitations []packngo.Invitation) diag.Diagnostics {
+func (m *ResourceModel) parse(ctx context.Context, member *member) diag.Diagnostics {
+	var diags diag.Diagnostics
+
+	if member.isMember() {
+		projectsList, diag := types.SetValueFrom(ctx, types.StringType, member.Member.Projects)
+		if diag.HasError() {
+			return diag
+		}
+		m.ProjectsIDs = projectsList
+		m.State = types.StringValue("active")
+
+		rolesList, _ := types.SetValueFrom(ctx, types.StringType, member.Member.Roles)
```

Comment:
There's a potential error here that we should also return instead of ignoring it